librz/bin/pdb/tpi.c

```diff
@@ -28,7 +70,7 @@ static bool is_simple_type(int idx) {
  *  This can be made smarter by using the masks
  *  and splitting it on 2 parts, 1 mode, 1 type
  */
-static STypeInfo parse_simple_type(ut32 idx) {
+RZ_API STypeInfo parse_simple_type(ut32 idx) {
```

Comment:
Is this necessary to be the API? Or `RZ_IPI` would be enough? If it should be an API we have a particular naming convention for that.